pymtl3/datatypes/test/bit_structs_test.py

```diff
@@ -0,0 +1,92 @@
+"""
+==========================================================================
+bit_structs_test.py
+==========================================================================
+Test cases for bit_structs.
+
+Author : Yanghui Ou
+  Date : July 27, 2019
+"""
+from __future__ import print_function
+
+from ..bit_structs import _IS_BIT_STRUCT, bit_struct, field, mk_bit_struct
```

Comment:
do you need to import _IS_BIT_STRUCT here?

Reply:
oh I wanted to use it to test whether a class is a bit struct but forgot to implement the test. I'll add more tests.